src/app/containers/Article/index.jsx

```diff
@@ -34,6 +34,7 @@ const metadataProps = (amp, config, id, metadata, promo, service) => {
     locale: config.locale,
     metaTags: allTags(metadata.tags),
     opengraphSiteName: config.opengraphSiteName,
+    serviceName: config.opengraphSiteName,
```

Comment:
We already pass `config.opengraphSiteName` in as `opengraphSiteName` in the line above. We should use that, it can be re-named to something more generic.